src/main/java/com/kodilla/ecommercee/UserController.java

```diff
@@ -10,26 +9,20 @@
 public class UserController {
     private long key;
```

Comment:
Domyślny scope kontrolerów restowych w Springu to SINGLETON. W mojej ocenie nie powinno się trzymać takich wartości jako atrybuty obiektów, gdyż może to powodować dziwne zachowanie przy wysyłaniu wielu requestów jednocześnie. A w takim środowisku na produkcji będzie pracowała nasza aplikacja. 

https://stackoverflow.com/a/33142442